### PR TITLE
Collateral Omnibus 7

### PR DESCRIPTION
**Pull request by ChandlerDeKock**

### CODE REVIEW

File: Omnibus 7.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+To accomplish this upgrade, the following needs to be added to the collateral currency whitelist introduced in UMIP-8.
+
+### PERP 
+-   The **PERP** Ethereum [token address](https://etherscan.io/token/0xbc396689893d065f41bc2c6ecbee5e0085233447): 0xbc396689893d065f41bc2c6ecbee5e0085233447 
+-   A final fee of **28 PERP tokens** needs to be added in the Store contract.
+
+### GRO
+-   The **GRO** Ethereum [token address](https://etherscan.io/address/0x3ec8798b81485a254928b70cda1cf0a2bb0b74d7): 0x3ec8798b81485a254928b70cda1cf0a2bb0b74d7 
+-   A final fee of **10 000 GRO tokens** needs to be added in the Store contract.
```
Comment (Reinis-FRP) [inline Omnibus 7.md:26]: If pricing on Balancer is legitimate then 1 GRO seems to be $8.7, so final fee valued at $87k might be too high

File: Omnibus 7.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+### GRO
+-   The **GRO** Ethereum [token address](https://etherscan.io/address/0x3ec8798b81485a254928b70cda1cf0a2bb0b74d7): 0x3ec8798b81485a254928b70cda1cf0a2bb0b74d7 
+-   A final fee of **10 000 GRO tokens** needs to be added in the Store contract. 
+
+### POOL
+-   The **POOL** Polygon [token address](https://polygonscan.com/address/0x25788a1a171ec66da6502f9975a15b609ff54cf6): 0x25788a1a171ec66da6502f9975a15b609ff54cf6 
+-   A final fee of **45** needs to be added in the Store contract. 
+## Rationale
+
+This store fee was chosen as it is approximately equivalent to $400 in line with other collateral currencies as determined by using the average price observed on Coinmarketcap. For the GRO token, the fee is set above the expected value of $400 since there is no live trading price.
```
Comment (Reinis-FRP) [inline Omnibus 7.md:33]: GRO Balancer pool got funded on 27 September 2021: https://etherscan.io/tx/0x75493a80d022d99cbab802094983698027afe5053330d2e598088396290f61bf

**Comment by inalittlewhile** [comment]
Verified that the contract address (0x3ec8798b81485a254928b70cda1cf0a2bb0b74d7) matches the GRO DAO Token on both Etherscan and CoinGecko. 

Liquidity was provided on Uniswap V2 at tx:
[https://etherscan.io/tx/0x3a3cd7ebbafa7b95770130064b2f7ebed3c8679126ce1e7dadd76368a365a2f8](https://etherscan.io/tx/0x3a3cd7ebbafa7b95770130064b2f7ebed3c8679126ce1e7dadd76368a365a2f8) 

This GRO/USDC LP holds 8.7426% of distributed GRO and [shows a price of 13.5319 USDC per GRO](https://info.uniswap.org/#/pools/0xf2d844e2db1f2a3ca2db62aba5e492f8b032b817) 

Note: Per Etherscan, two addresses hold roughly 70% of total distributed GRO. One at 38.7568% and one at 31.2815%. GRO is distributed between a total of 1055 addresses.

File: Omnibus 7.md (hunk 1 above)
Comment (mhairimca) [reply Omnibus 7.md:26]: Current [price on coingecko](https://www.coingecko.com/en/coins/gro-dao-token) seems to be $16.35, which would suggest a final fee of around 25 GRO.

**Pull request merged**